### DISCUSSION TITLE
%E8%B5%84%E6%BA%90%E5%88%86%E4%BA%AB/book/

### DISCUSSION THREAD

**Discussion by giscus** [ForsakenDelusion/AUST-CS-Plan#5] [Announcements]
# %E8%B5%84%E6%BA%90%E5%88%86%E4%BA%AB/book/

AUST计算机求生指南

https://aust-cs-plan.forsakendelusion.online/%E8%B5%84%E6%BA%90%E5%88%86%E4%BA%AB/book/

**Comment by bruh3506** [comment]
🐂

**Reply by ForsakenDelusion** [reply]
🤖
